scripts/metrics/common/config.py

```diff
@@ -0,0 +1,79 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Config file utilities for metrics scripts
+"""
+
+import os
+import sys
+
+import yaml
+
+from common.exception import MetricsExn
+
+config = {}
+
+
+class ConfigExn(MetricsExn):
+    pass
+
+
+def load_config(local_path, override=None):
+    global config
```

Comment:
I prefer that the read_config() uses only the same global singleton to prevent the mishap of having multiple/different config objects. We can pass the config object around in the script if it's strongly preferred, but imo this way is cleaner.

Reply:
I think it's cleaner not to provide an optional config file with overrides, and simply make a data transformer that expects a JUnit-style XML and condenses the data into CSVs with named columns.

Reply:
Removed config util file